catalog/core/catalog-core-camelcomponent/pom.xml

```diff
@@ -100,6 +100,10 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast</artifactId>
```

Comment:
I would prefer not to switch to EHCache this close to 2.10, though we should look into HA for this functionality after it is cut.